app/layout/header.phtml

```diff
@@ -15,7 +15,7 @@
 		<?php if (FreshRSS_Auth::hasAccess() || FreshRSS_Context::$system_conf->allow_anonymous) { ?>
 		<form action="<?= _url('index', 'index') ?>" method="get">
 			<div class="stick">
-				<input type="search" name="search" id="search" class="extend"
+				<input type="search" name="search" id="search"
```

Comment:
I'd make it class=long.

Btw, isn't there also `.header .item.search input:focus` in some of the themes or was that changed in the meantime?

Reply:
fixed